setup.py

```diff
@@ -11,8 +17,10 @@
       maintainer_email='mrocklin@gmail.com',
       license='BSD',
       keywords='',
-      packages=['cachey'],
+      packages=['cachey', 'cachey.tests'],
```

Comment:
Without __init__.py setup.py will not install the files to the dest location.

I am not sure if setup.py instructs pytest to find files in the installation destination or from the current source.

I do think dask adds these test packages in its setup.py.

Reply:
Just to be clear, `cachey.tests` does not need to be listed in `packages` as well.

There may be other changes needed for pytest, but this comment is only related to packaging specifically.

Reply:
Perhaps. I suppose installing the tests would allow running tests from an installation without referring to the source code; and that's why dask has it in:

https://github.com/dask/dask/blob/master/setup.py#L49

Reply:
@jakirkham , any thought on the last point?

Reply:
Sorry missed this it seems.

They will get installed once we have the `__init__.py` added in this PR. Listing them in `packages` won't affect whether they are included either way. So would suggest keeping things simple.

```suggestion
      packages=['cachey'],
```